WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -558,11 +558,14 @@
           regionalSingleRecord: "NAR",
           regionalAverageRecord: "",
         )
+        expect(competition.results_posted_at).to be nil
         get :post_results, id: competition
```

Comment:
Yes, this is my fault. I did it in order to get announcement posting from PHP to work. See https://github.com/cubing/worldcubeassociation.org/issues/161